django/contrib/auth/views.py

```diff
@@ -438,6 +442,10 @@ class PasswordResetConfirmView(PasswordContextMixin, FormView):
     title = _('Enter new password')
     token_generator = default_token_generator
 
+    def _build_redirect_url(self):
```

Comment:
Move a unit of logic into its own function. This part has been simplified over time, after successive reviews.